lib/tpm2_tcti_ldr.h

```diff
@@ -34,8 +34,8 @@
  * Loads a TCTI from a friendly name, library name, or path.
  * For example
  *  friendly:     path = tabrmd
- *  library name: path = libtcti-socket.so
- *  full path:    path = /home/user/lib/libtcti-custom.so
+ *  library name: path = libtss2-tcti-socket.so
```

Comment:
Wasn't the socket TCTI recently renamed to mssim?

Reply:
Ill update this after we pull this in.